Mail/Views/Thread/WebView/WebViewModel.swift

```diff
@@ -64,13 +65,36 @@ final class WebViewModel: NSObject, ObservableObject {
         loadScripts(configuration: webView.configuration)
     }
 
-    func loadBody(presentableBody: PresentableBody, blockRemoteContent: Bool) async throws -> LoadResult {
+    func loadBody(presentableBody: PresentableBody, blockRemoteContent: Bool, messageUid: String) async throws -> LoadResult {
         var messageBody = showBlockQuote ? presentableBody.body?.value : presentableBody.compactBody
 
+        if messageBody != nil, let subBodies = presentableBody.body?.subBody {
+            messageBody! += formatSubBodyContent(subBodies: subBodies, messageUid: messageUid)
+        }
+
         let loadResult = await loadHTMLString(value: messageBody, blockRemoteContent: blockRemoteContent)
         return loadResult
     }
 
+    private func formatSubBodyContent(subBodies: List<SubBody>, messageUid: String) -> String {
+        var subBodiesContent = ""
+        for subBody in subBodies {
```

Comment:
It is great like this. If you want to have fun and try to remove the mutable `subBodiesContent` I think you can achieve it with a `reduce`. That said, reduce can be harder to comprehend so I'm on the fence with using it.

Reply:
That's a good idea !